### PR TITLE
Add setup function for auto-deploy of Studio Flow

### PR DESCRIPTION
**Pull request by cweems**
This pull request adds a function that deploys the Studio Flow with the correct Twilio Function configuration. It then updates the phone number with the Studio Flow's webhook URL.

Note: the function is currently configured to search specifically for "dev-environment" as the environment, but it appears that the Code Exchange will create the serverless environment as the name of the serverless project as specified in package.json (vaccine-standby-list). We should deploy a new serverless environment and update L38 to test.

### CODE REVIEW

File: vaccine-standby-list/functions/hello-messaging.protected.js [hunk 1]
```diff
@@ -1,5 +0,0 @@
-exports.handler = function(context, event, callback) {
```
Comment (jme783) [inline vaccine-standby-list/functions/hello-messaging.protected.js:1]: Thanks for removing this unused file

File: vaccine-standby-list/functions/setup.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
+let flowDefinition = require('../sms-studio-flow.json');
+
+exports.handler = async function(context, event, callback) {
+
+	const client = context.getTwilioClient();
```
Comment (jme783) [inline vaccine-standby-list/functions/setup.js]: Indentation looks slightly off here

File: vaccine-standby-list/functions/setup.js [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+                })
+                .catch(err => reject(err));;
+        })
+    }
+
+    function updatePhoneNumberWebhook(studioWebhook, numberSid) {
+        return new Promise((resolve, reject) => {
+            client.incomingPhoneNumbers(numberSid)
+                .update({
+                    voiceUrl: studioWebhook
```
Comment (jme783) [inline vaccine-standby-list/functions/setup.js]: @cweems shouldn't this be the sms URL not voice URL?

**Review by jme783 — COMMENTED** [review]
One thing to call out in particular is updating the phone number config. Right now it's set to update `voiceUrl` — wondering if it should be `smsUrl`?

File: vaccine-standby-list/functions/setup.js (hunk 2 above)
Comment (cweems) [reply vaccine-standby-list/functions/setup.js]: mixed tabs and spaces!

**Review by jme783 — APPROVED** [review]
Noice

**Pull request merged**